Sources/Controllers/ConfigureScreen/ConfigureScreenViewController.swift

```diff
@@ -12,6 +12,7 @@ import Foundation
 @objc(OAWidgetStateDelegate)
 protocol WidgetStateDelegate: AnyObject {
     func onWidgetStateChanged()
+    @objc optional func updateQuickActionData()
```

Comment:
updateQuickActionData is not part of WidgetState

Reply:
Could you create a protocol with actions for OAQuickActionListViewController, or use a callback (weak var delegate: OAQuickActionDelegate?)